chat/room.go

```diff
@@ -200,11 +206,15 @@ func (r *Room) Leave(u *message.User) error {
 }
 
 // Rename member with a new identity. This will not call rename on the member.
-func (r *Room) Rename(oldID string, u message.Identifier) error {
+func (r *Room) Rename(oldID string, u *Member) error {
```

Comment:
Can we avoid changing this public function signature? I believe the host has everything it needs to look up users by ID?

Reply:
The Identifier interface technically doesn't fulfill the needs of checkName(). But... this could probably be fixed best with a signature like `func ( nick string, fingerprint string error` instead, so that it's up to the caller to handle the lookups and type assertions. I'll try that.